nemo_curator/scripts/fuzzy_deduplication/README.md

```diff
@@ -1,99 +1,2 @@
-          --buckets-per-shuffle 1 `#Value b/w [1-num_bands]. Higher is better but might lead to oom` \
-          --log-dir ./
-          # --scheduler-file /path/to/file.json
-    ```
-3. Jaccard Map Buckets
-    - Input: Buckets.parquet + Data Dir
-    - Output: anchor_docs_with_bk.parquet
-    - Example call:
-    ```bash
-        # same as `python map_buckets.py`
-        jaccard_map_buckets \
-          --input-data-dirs /path/to/jsonl/dir1 /path/to/jsonl/dir2 \
-          --input-bucket-dir /path/to/dedup_output/_buckets.parquet \
-          --output-dir /path/to/dedup_output \
-          --input-json-text-field text_column_name \
-          --input-json-id-field id_column_name \
-          # --scheduler-file /path/to/file.json
-    ```
-4. Jaccard Shuffle
-    - Input: anchor_docs_with_bk.parquet + Data Dir
-    - Output: shuffled_docs.parquet
-    - Example call:
-    ```bash
-        # same as `python jaccard_shuffle.py`
-        jaccard_shuffle \
-          --input-data-dirs /path/to/jsonl/dir1 /path/to/jsonl/dir2 \
-          --input-bucket-mapping-dir /path/to/dedup_output/anchor_docs_with_bk.parquet \
-          --output-dir /path/to/dedup_output \
-          --input-json-text-field text_column_name \
-          --input-json-id-field id_column_name \
-          # --scheduler-file /path/to/file.json
-    ```
-5. Jaccard compute
-    - Input: Shuffled docs.parquet
-    - Output: jaccard_similarity_results.parquet
-    - Example call:
-    ```bash
-        # same as `python jaccard_compute.py`
-        jaccard_compute \
-          --shuffled-docs-path /path/to/dedup_output/shuffled_docs.parquet \
-          --output-dir /path/to/dedup_output \
-          --ngram-size char_ngram_size_for_similarity \
-          # --scheduler-file /path/to/file.json
-    ```
-6. Connected Components
-    - Input: jaccard_similarity_results.parquet
-    - Output: connected_components.parquet
-    - Example call:
-    ```bash
-        # same as `python connected_components.py`
-        gpu_connected_component \
-          --jaccard-pairs_path /path/to/dedup_output/jaccard_similarity_results.parquet \
-          --output-dir /path/to/dedup_output \
-          --cache-dir /path/to/cc_cache \
-          --jaccard-threshold 0.8
-          # --scheduler-file /path/to/file.json
-    ```
-
-> [!TIP]
-> When using these scripts in a multi-node environment (like Slurm, K8's etc.) it is recommended to start up a Dask cluster prior to execution and connect to the existing cluster via the `--scheduler-address` or `--scheduler-file` flag.
-> Use the `--help` flag to view all possible CLI options for the scripts and details on what they do.
+> The up to date documentation on running the Fuzzy Deduplication scripts can be found in the [NeMo-User guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/datacuration/gpudeduplication.html#id4). It is recommended to use the Python API directly rather than the CLI scripts for most cases.
```

Comment:
```suggestion
> The up to date documentation on running the fuzzy deduplication scripts can be found in the [NeMo Curator User Guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/datacuration/gpudeduplication.html#id4). It is recommended to use the Python API directly rather than the CLI scripts for most cases.
```